templates/includes/sidebar.html

```diff
@@ -0,0 +1,20 @@
+<ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: {{ user.username }}</li>
+    <form action="{% url 'logout' %}" method="POST">
+      {% csrf_token %}
+      <button type="submit" class="btn btn-danger">Logout</button>
+    </form>
+
+  {% else %}
+    <form action="{% url 'login' %}" method="POST">
+      {% csrf_token %}
+      <button type="submit" class="btn btn-primary">Login</button>
+    </form>
+
+  {% endif %}
+{#  <li><a href="{% url "taxi:index" %}">Home page</a></li>#}
+{#  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>#}
+{#  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>#}
+{#  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>#}
```

Comment:
These lines of code are commented out and do not contribute to the functionality of your application. If these lines are not necessary, consider removing them to improve the readability and cleanliness of your code.